### PR TITLE
chore: bumped wasmd version to v0.28.0

### PR DESCRIPTION
**Pull request by swelf19**
implements https://p2pvalidator.atlassian.net/jira/software/projects/NTRN/boards/121?selectedIssue=NTRN-146

wasmd fork was updated to v0.28.0 and event filtering was disabled https://github.com/neutron-org/wasmd/commit/600e1f7c39a230c513301be1f4ee4a4cec87392c

1) Wrapper for ibc transfer module is updated, now it wraps module on a very top level with redefining the rpc server
2) `transfer.Transfer` and `interchaintxs.SubmitTx` return responds with the sequence_id

test with updated contracts: https://github.com/neutron-org/neutron-contracts/pull/25

### CODE REVIEW

**Review by foxpy — CHANGES_REQUESTED** [review]
This breaks `test:simple` in our integration tests framework

**Comment by swelf19** [comment]
> This breaks `test:simple` in our integration tests framework

breaking changes in the commit  https://github.com/CosmWasm/wasmd/commit/ee44cc06d0a6c8a35c5657cb765f6c6ba39902fa that part of 0.28.0 release

**Comment by foxpy** [comment]
> breaking changes in the commit [CosmWasm/wasmd@ee44cc0](https://github.com/CosmWasm/wasmd/commit/ee44cc06d0a6c8a35c5657cb765f6c6ba39902fa) that part of 0.28.0 release

Are we going to revert this commit (sounds dangerous) or try to build stuff differently without relying on events somehow?

**Comment by pr0n00gler** [comment]
> > breaking changes in the commit [CosmWasm/wasmd@ee44cc0](https://github.com/CosmWasm/wasmd/commit/ee44cc06d0a6c8a35c5657cb765f6c6ba39902fa) that part of 0.28.0 release
> 
> Are we going to revert this commit (sounds dangerous) or try to build stuff differently without relying on events somehow?

We can't just revert it because it's very important security commit. We're trying to deal with the problems without relying on events.

File: wasmbinding/bindings/msg.go [hunk 1]
```diff
@@ -29,7 +29,12 @@ type SubmitTx struct {
 }
 
 // SubmitTxResponse holds response from SubmitTx.
-type SubmitTxResponse struct{}
+type SubmitTxResponse struct {
+	// SequenceId is a channel's sequence_id for outgoing ibc packet. Unique per a channel.
+	SequenceId uint64 `json:"sequence_id"`
+	// Channek is a src channel on neutron side trasaction was submitted from
```
Comment (foxpy) [inline wasmbinding/bindings/msg.go]:
```suggestion
	// Channel is a src channel on neutron side transaction was submitted from
```

**Review by foxpy — CHANGES_REQUESTED** [review]

**Review by foxpy — DISMISSED** [review]

**Review by foxpy — APPROVED** [review]

File: proto/interchaintxs/v1/tx.proto [hunk 2]
```diff
@@ -60,4 +60,9 @@ message MsgSubmitTx {
 }
 
 // MsgSubmitTxResponse defines the response for Msg/SubmitTx
-message MsgSubmitTxResponse {}
\ No newline at end of file
+message MsgSubmitTxResponse {
+  // channel's sequence_id for outgoing ibc packet. Unique per a channel.
+  uint64 sequence_id = 1;
+  // channel src channel on neutron side trasaction was submitted from
+  string channel = 2;
```
Comment (zavgorodnii) [inline proto/interchaintxs/v1/tx.proto:67]: channel_id?

**Review by zavgorodnii — APPROVED** [review]

**Pull request merged**